tests/Operation/FindAndModifyFunctionalTest.php

```diff
@@ -92,6 +93,23 @@ public function testHintOptionAndUnacknowledgedWriteConcernUnsupportedClientSide
         $operation->execute($this->getPrimaryServer());
     }
 
+    public function testFindAndModifyReportedWriteConcernError(): void
```

Comment:
This test should be restricted to replica sets, as the exception message will differ on standalone servers (see failed test).

Reply:
done